app/routes/quotes/QuotesRoute.js

```diff
@@ -12,40 +11,39 @@ import prepare from 'app/utils/prepare';
 import { selectSortedQuotes } from 'app/reducers/quotes';
 import { LoginPage } from 'app/components/LoginForm';
 import replaceUnlessLoggedIn from 'app/utils/replaceUnlessLoggedIn';
-import { selectPagination } from '../../reducers/selectors';
+import { selectPaginationNext } from 'app/reducers/selectors';
 import { addReaction, deleteReaction } from 'app/actions/ReactionActions';
 import { selectEmojis } from 'app/reducers/emojis';
 import { fetchEmojis } from 'app/actions/EmojiActions';
 import qs from 'qs';
 
+const qsParamsParser = (search) => ({
+  approved: qs.parse(search, { ignoreQueryPrefix: true }).approved || 'true',
+});
```

Comment:
I kind of like having the default `true` in the selector (as it was before) and just pass `undefined` in the query here tbh. It just makes it more consistent, but not really a big deal. As it is now, if you omit `approved` in the `props` to `sortedSelector` you'd just get `[]`. Which could be okay, but I think defaulting to approved makes more sense.

Reply:
Switched this to a bool to make it more consistent ref. @ivarnakken's comment. Passing an empty "approved" will select both approved and unapproved, ref how pagination works, and creating the default in the selector would be a bit awkward imo. and will not be consistent with the other use of paginationNext..